pipelined/bevy_pbr2/Cargo.toml

```diff
@@ -12,6 +12,10 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT"
 keywords = ["bevy"]
 
+[features]
+default = []
```

Comment:
```suggestion
```
Is there a reason to add an empty default?

Reply:
none, it's just that I'm used to it